Kiosk/App/Models/SaleArtworkViewModel.swift

```diff
@@ -19,19 +19,33 @@ extension SaleArtworkViewModel {
 
     var estimateString: String {
         // Default to estimateCents
-        switch (saleArtwork.estimateCents, saleArtwork.lowEstimateCents, saleArtwork.highEstimateCents) {
+        switch (
+            saleArtwork.estimateCents,
+            saleArtwork.lowEstimateCents,
+            saleArtwork.highEstimateCents
+        ){
         // Default to estimateCents.
         case (.some(let estimateCents), _, _):
-            let dollars = centsToPresentableDollarsString(estimateCents, currencySymbol: saleArtwork.currencySymbol)
+            let dollars = centsToPresentableDollarsString(
+                estimateCents,
+                currencySymbol: saleArtwork.currencySymbol
+            )
             return "Estimate: \(dollars)"
 
-        // Try to extract non-nil low/high estimates.
+            // Try to extract non-nil low/high estimates.
```

Comment:
Probably a tough one to change.

Reply:
Mmm, yeah. Not a huge deal though, could easily move the comment into the case.